kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/security/auth/ResourceType.java

```diff
@@ -35,6 +35,11 @@
      */
     TOPIC((byte) 2),
 
+    /**
+     * A Pulsar tenant.
+     */
+    TENANT((byte) 3),
+
```

Comment:
The code should be in order. Can you please revise it by the way?
```java
/**
   * A Pulsar topic.
   */
  TOPIC((byte) 1),

  /**
   * A Pulsar tenant.
   */
  TENANT((byte) 2),
```

Reply:
sure